reference/toolstacks/python/README.md

```diff
@@ -0,0 +1,174 @@
+# Python
+
+Platform.sh supports deploying Python applications, your application can use
```

Comment:
`-` `, your application can use WSGI-based`
`+` `. Your application can use the WSGI-based`